chain/syncer_integration_test.go

```diff
@@ -205,7 +205,7 @@ func (isb *integrationStateBuilder) ComputeState(prev cid.Cid, blocksMessages []
 		return genRoot, nil
 	}
 	// setup fork with state we associate with more power
-	if len(blocksMessages[0]) > 0 {
+	if len(secpMessages[0]) > 0 {
```

Comment:
This is only ok because it's test code. I think we should leave a comment stating the assumption of secp messages made here. Or just generalise it now to handle possibly-empty secp messages but also check bls messages.